packages/foundry/src/vault/adapter/beefy/BeefyAdapter.sol

```diff
@@ -62,6 +64,11 @@ contract BeefyAdapter is AdapterBase, WithRewards {
 
     beefyBalanceCheck = IBeefyBalanceCheck(_beefyBooster == address(0) ? _beefyVault : _beefyBooster);
 
+    IBeefyStrat strat = IBeefyStrat(beefyVault.strategy());
```

Comment:
Technically beefy could change the strategy but i have never seen that happen. So im still contemplating if we should have the try catch in each preview call or just here once.

Reply:
This is something that could change if the underlying protocol moves to a different address. I've seen it happen a few times that required strategy migration.